examples/out-of-tree/cloud-controller-manager.yaml

```diff
@@ -129,68 +129,77 @@ subjects:
     kind: User
     name: cloud-controller-manager
 ---
-apiVersion: v1
-kind: Pod
+apiVersion: apps/v1
+kind: Deployment
 metadata:
   name: cloud-controller-manager
   namespace: kube-system
   labels:
-    tier: control-plane
     component: cloud-controller-manager
 spec:
-  priorityClassName: system-node-critical
-  hostNetwork: true
-  serviceAccountName: cloud-controller-manager
-  tolerations:
-    - key: node-role.kubernetes.io/master
-      effect: NoSchedule
-  containers:
-    - name: cloud-controller-manager
-      image: mcr.microsoft.com/oss/kubernetes/azure-cloud-controller-manager:v1.23.5
-      imagePullPolicy: IfNotPresent
-      command: ["cloud-controller-manager"]
-      args:
-        - "--allocate-node-cidrs=true" # "false" for Azure CNI and "true" for other network plugins
-        - "--cloud-config=/etc/kubernetes/azure.json"
-        - "--cloud-provider=azure"
-        - "--cluster-cidr=10.244.0.0/16"
-        - "--cluster-name=k8s"
-        - "--controllers=*,-cloud-node" # disable cloud-node controller
-        - "--configure-cloud-routes=true" # "false" for Azure CNI and "true" for other network plugins
-        - "--leader-elect=true"
-        - "--route-reconciliation-period=10s"
-        - "--v=2"
-        - "--port=10267"
-      resources:
-        requests:
-          cpu: 100m
-          memory: 128Mi
-        limits:
-          cpu: "4"
-          memory: 2Gi
-      livenessProbe:
-        httpGet:
-          path: /healthz
-          port: 10267
-        initialDelaySeconds: 20
-        periodSeconds: 10
-        timeoutSeconds: 5
-      volumeMounts:
+  replicas: 1
+  selector:
+    matchLabels:
+      component: cloud-controller-manager
+  template:
+    metadata:
+      labels:
+        tier: control-plane
+        component: cloud-controller-manager
+    spec:
+      priorityClassName: system-node-critical
+      hostNetwork: true
+      serviceAccountName: cloud-controller-manager
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
+      containers:
+        - name: cloud-controller-manager
+          image: mcr.microsoft.com/oss/kubernetes/azure-cloud-controller-manager:v1.23.5
+          imagePullPolicy: IfNotPresent
+          command: ["cloud-controller-manager"]
+          args:
+            - "--allocate-node-cidrs=true" # "false" for Azure CNI and "true" for other network plugins
+            - "--cloud-config=/etc/kubernetes/azure.json"
+            - "--cloud-provider=azure"
+            - "--cluster-cidr=10.244.0.0/16"
+            - "--cluster-name=k8s"
+            - "--controllers=*,-cloud-node" # disable cloud-node controller
+            - "--configure-cloud-routes=true" # "false" for Azure CNI and "true" for other network plugins
+            - "--leader-elect=true"
+            - "--route-reconciliation-period=10s"
+            - "--v=2"
```

Comment:
can we make v=4?

Reply:
v=5 by default.

Reply:
I think 4 is enough for debugging.

Reply:
Let's use 4 for now and update in the future if needed.